cmd/zoekt-archive-index/main.go

```diff
@@ -152,6 +180,11 @@ func do(opts Options, bopts build.Options) error {
 	}
 	defer a.Close()
 
+	ig, err := newIgnoreMatcher(opts.Archive, ignoreOptions{Strip: opts.Strip})
```

Comment:
I didn't add a flag to `zoekt-archive-index` (and consequently neither to `zoekt-sourcegraph-indexserver`) because I assume we want to switch the feature on for all customers. However, repos without a `sourcegraphignore` would pay a small performance fee during indexing.  Is there a better way to check for the existence of a `sourcegraphignore` file without looping through all files?

Reply:
This comment is not relevant anymore, because we decided to add support for `sourcegraphignore` to `zoekt-git-index` instead of `zoekt-archive-index`.